src/spectral/rulesets/ibm-oas/response-example-provided.js

```diff
@@ -0,0 +1,18 @@
+module.exports = function(response) {
+  if (!responseLevelExamples(response) && !schemaLevelExample(response)) {
+    return [
+      {
+        message:
+          'To generate good documenation, should provide response example(s) inside of or at the the same level as the schema'
```

Comment:
This is a little long, but it's probably fine. I will make some minor suggestions:

```suggestion
          'To generate good documentation, provide response example(s) inside of, or at the the same level as, the schema'
```

Reply:
How about:
```suggestion
          'Response bodies should include an example response'
```

I don't think we need "To generate good documentation" -- that is the goal of many of our rules (like missing descriptions) but we don't explicitly state that in the message.

And I don't think the message needs to say where the example should be specified -- we say that in the documentation of the rule.